### ISSUE TITLE
Why is this value of "training/unsup/loss" increasing?

### ISSUE THREAD

**Issue by fengyunsanchijian** [google-research/uda#73]
Hi.
Why is this value of "training/unsup/loss" increasing? I think this value should be lowered during training.
[The unsup loss curve](https://ftp.bmp.ovh/imgs/2019/12/4c2a172030b165dd.jpg)
===========================
Parameters:
INFO:tensorflow:sup batch size: 8
INFO:tensorflow:sup batch size: 8
INFO:tensorflow:unsup batch size: 8
INFO:tensorflow:total sample in a batch: 24
===========================
Other parameters are the default.
Thanks

**Comment by Guohai93** [comment]
I meet the same situation.

**Comment by airship-explorer** [comment]
> # Hi.
> Why is this value of "training/unsup/loss" increasing? I think this value should be lowered during training.
> [The unsup loss curve](https://ftp.bmp.ovh/imgs/2019/12/4c2a172030b165dd.jpg)
> # Parameters:
> INFO:tensorflow:sup batch size: 8
> INFO:tensorflow:sup batch size: 8
> INFO:tensorflow:unsup batch size: 8
> INFO:tensorflow:total sample in a batch: 24
> Other parameters are the default.
> Thanks

I meet the same situation.
